src/commands/synthetics/utils/public.ts

```diff
@@ -78,24 +79,29 @@ export const getOverriddenConfig = (
   test: Test,
   publicId: string,
   reporter: MainReporter,
-  config?: UserConfigOverride
+  // TODO SYNTH-12989: Clean up deprecated `config` in favor of `testOverrides`
+  config?: UserConfigOverride,
+  testOverrides?: UserConfigOverride
```

Comment:
This whole file is publicly exported so had to set up compatibility stuff for each function

Reply:
This doesn't make sense: the content of the object stays the same, and the type as well. Since you would only change the name of the parameter, this is not a breaking change from TypeScript's and even JavaScript's point of view.

This function can stay untouched. Only our usages of it should be updated, and you already did it 👍